### ISSUE TITLE
Lib.d.ts `addEventListener` should specify `ClipboardEvent` payload for relevant events

### ISSUE THREAD

**Issue by NoelAbrahams** [microsoft/TypeScript#5363]
Hi,

**Version**: 1.6

The event payload for the following should be [`ClipboardEvent`](https://developer.mozilla.org/en-US/docs/Web/API/ClipboardEvent)

``` typescript
addEventListener(type: "beforecopy", listener: (ev: DragEvent) => any, useCapture?: boolean): void;
addEventListener(type: "beforecut", listener: (ev: DragEvent) => any, useCapture?: boolean): void;
addEventListener(type: "beforepaste", listener: (ev: DragEvent) => any, useCapture?: boolean): void;
addEventListener(type: "copy", listener: (ev: DragEvent) => any, useCapture?: boolean): void;
addEventListener(type: "cut", listener: (ev: DragEvent) => any, useCapture?: boolean): void;
addEventListener(type: "paste", listener: (ev: DragEvent) => any, useCapture?: boolean): void;
```

The `ClipboardEvent` type is defined but not used anywhere:

``` typescript
interface ClipboardEvent extends Event {
    clipboardData: DataTransfer;
}
```

**Comment by mhegazy** [comment]
PRs are welcomed. here is some documentation for adding new types to lib.d.ts: https://github.com/Microsoft/TypeScript/blob/master/CONTRIBUTING.md#contributing-libdts-fixes

**Comment by saschanaz** [comment]
This can be closed as now ClipboardEvent is being used.

**Comment by jakebailey** [comment]
Just looking at old issues; this one's been fixed for a while (since #7619), per the above.
